bin/start-func-test-server.js

```diff
@@ -1,198 +0,0 @@
-/* @flow */
```

Comment:
this file was only for disco?  (couldn't see anything particularly disco related in it, but I don't know what it any of it does 😄 )

Reply:
It was only used for the ui tests on travis-ci, which covered the discopane only